patches/ffmpeg/build/build.sh

```diff
@@ -6,16 +6,20 @@ build_type="$3"
 enable_gpl="$4"
 
 extra_cflags=""
-extra_flags=""
+
+declare -a extra_flags
```

Comment:
since we are going with arrays, I would use the array for the entire command. Eg.

```
declare -a configure_cmd
declare -i idx=0

configure_cmd[idx++]="./configure"
configure_cmd[idx++]="--toolchain=msvc"
...
```

Reply:
sounds good